src/contracts/GPv2Settlement.sol

```diff
@@ -133,6 +133,21 @@ contract GPv2Settlement {
     }
 
     // solhint-enable
+    modifier onlyOwner {
```

Comment:
nit: could you add a newline between the comment and the modifier, otherwise it looks like the `//solhint-enable` is for the modifier, but it is not.